client/views/connection_list.tpl

```diff
@@ -0,0 +1,11 @@
+<p class="clearfix">
+	{{#if id includeZero=true}}
+		<button class="btn pull-right" id="connection_{{id}}" value="{{socketId}}">Disconnect</button>
+	{{/if}}
+
+	<strong>{{hostname}}</strong>
+	<br>
+	<a class="" href="https://geoiptool.com/?ip={{ip}}">{{ip}}</a>
```

Comment:
There is no way we can ever have this in-repo. Note that it's not fetching anything, just adding a link to it. I'd rather have this than nothing.